src/pyspiffe/workloadapi/default_workload_api_client.py

```diff
@@ -101,3 +120,24 @@ def validate_jwt_svid(self, token: str, audience: str) -> JwtSvid:
         """
 
         pass
+
+    def get_spiffe_endpoint_socket(self) -> str:
+        """Returns the spiffe endpoint socket config for this WorkloadApiClient.
+
+        Returns:
+            str: spiffe endpoint socket configuration value.
+        """
+
+        return self._config.spiffe_endpoint_socket
+
+    def _get_spiffe_grpc_channel(self):
+        grpc_insecure_channel = grpc.insecure_channel(
+            self._config.spiffe_endpoint_socket
+        )
+        spiffe_client_interceptor = (
+            header_manipulator_client_interceptor.header_adder_interceptor(
+                _WORKLOAD_API_HEADER, 'true'
```

Comment:
Maybe `true` should be a constant as well? Although this won't ever change, as long as the header name is a constant, it seems like the value for it should be too?

Reply:
yes, will define a constant in the abstract class.